layout-management-webapps/src/main/resources/locale/portlet/layoutManagement/SiteNavigationPortlet_en.properties

```
@@ -71,6 +71,15 @@ siteNavigation.label.editElement=Edit element
 siteNavigation.label.cutNode=Cut node
 siteNavigation.label.pasteNode=Paste node
 siteNavigation.label.pasteNode.error=This node name already exists.
-
-
-
+siteNavigation.label.node=Node
+siteNavigation.label.actions=Actions
+siteNavigation.label.nodeType=Type
+siteNavigation.label.visibility=Visibility
+siteNavigation.label.access=Access
+siteNavigation.label.visibility.temporal=Scheduled node
+siteNavigation.label.visibility.hidden=Hidden node
+siteNavigation.label.visibility.displayed=Displayed node
+siteNavigation.label.access.all=All space member can edit
+siteNavigation.label.access.specific=Only designated collaborators can edit
```

```suggestion
siteNavigation.label.access.all=All users can access
siteNavigation.label.access.specific=Only designated groups can access
```
labels to be confirmed with @mzorai